CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ Changes since v2.15
 
 - The OIDC configuration is now validated, and REMS refuses to start without a valid OIDC configuration. See the `:oidc-metadata-url` configuration variable and [configuration.md](docs/configuration.md). (#2519)
 
+- The navbar can be configured to show a logo image. When the `:navbar-logo-name` config is provided, the logo is shown in the navbar (top navigation menu). This logo also can be customized per language like the regular logo. (#2363)
```

Comment:
I think something that was lost in the chat was the idea that we should showcase this feature in the `example-theme/theme.edn`. So draw a new REMS small logo for the navbar (with localizations).

Reply:
Draw? you mean create a logo option for REMS default theme?

Reply:
I think we can also get by with just using `resources/public/img/rems_logo_*.png` as the navbar logo for now.

Reply:
right, so you mean add the logo to the default theme? Is that right?

Reply:
Yep, a navbar logo for the example theme.

Reply:
Just to be clear and make sure we are in agreement? I reeeeaaallly want to be done with the ticket.

Reply:
done

Reply:
Why do you only add it for Swedish though? There should be one for each language? 

Also I would just resize the logo image for the navbar (or make new ones, these are just for demo purposes anyway) and use separate files so that it is clear what is meant to happen and why.

Reply:
I was surprised by having the logo only for one languge too, but having it only for some languages makes it easy to preview both looks in dev, so it might work!

Reply:
yes, @opqdonut got my idea!

Reply:
I think it's ok for a demo for ourselves but not for code/config that we leave for others to read.

Reply:
What would you suggest could be the best approach then (also taking into account we need to demo both options)?
Have explicitly stated it in the theme file? create an alternative theme file?

Reply:
We have plenty of servers with the old config / other configs that we can ourselves use. As far as the future goes we want to advertise this new better option so it should be the complete default with sane values.

Reply:
"sane values"?

Reply:
Sane values for example config means that someone will get a good idea what are valid example values. I.e. the likes of which can be used in production.

Reply:
ok, done